docs/source/serving/env_vars.rst

```diff
@@ -3,6 +3,9 @@ Environment Variables
 
 vLLM uses the following environment variables to configure the system:
 
+.. warning::
+    Please note that ``VLLM_PORT`` and ``VLLM_HOST_IP`` set the port and ip for vLLM's **internal usage**. It is not the port and ip for the API server. If you use ``--host $VLLM_HOST_IP`` and ``--port $VLLM_PORT`` to start the API server, it will not work.
```

Comment:
Maybe say "these two variables should only be used when you experiencing hanging /crashing in vllm" ?

Reply:
Thanks for the comments. `VLLM_PORT` is introduced to follow some firewall rules. see https://github.com/vllm-project/vllm/pull/4914 .